docs/src/main/asciidoc/_cloud/Getting_Started_on_OpenShift.adoc

```diff
@@ -0,0 +1,150 @@
+```
+In the OpenShift sandbox, it is not possible to create a new project but, in case you are running your own OpenShift, the command to create a new project is:
+```
+oc new-project <project name>
+```
+### Helm Charts
+https://helm.sh/[Helm] provides an easy way to manage and share applications on Kubernetes and OpenShift. We will use it to deploy our example applications to OpenShift.
+
+##### Install Helm and Helm Charts
+See the instructions for installing Helm https://helm.sh/docs/intro/install/[here]. Helm is installed on your local system as a command, which we will use to install the WildFly Charts. Once Helm is installed, we can proceed to use it to install the Helm Charts in our OpenShift instance:
+```
+helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+```
+For more information on the WildFly Helm Charts, see this https://www.wildfly.org/news/2021/05/05/helm-charts-for-wildfly/[blog post] and https://docs.wildfly.org/wildfly-charts/[the WildFly Helm Charts documentation]
+
+## Example: To-do Quickstart
+The first example we will build and deploy to OpenShift is the https://github.com/wildfly/quickstart/tree/main/todo-backend[`todo-backend` quickstart]. This quickstart provides https://github.com/wildfly/quickstart/tree/main/todo-backend#run-the-backend-on-openshift[documentation] for installing via Helm, so only a brief version is detailed below.
+
+First, we need to provision a database instance, in this case PostgresSQL:
+```
+oc new-app postgresql-ephemeral \
+   -p DATABASE_SERVICE_NAME=todos-db \
+   -p POSTGRESQL_DATABASE=todos
+
+```
+Next we use the WildFly Helm Charts to build and deploy our quickstart:
+```
+helm install todo-backend -f https://raw.githubusercontent.com/wildfly/wildfly-charts/main/examples/todo-backend/todo-backend-bootable-jar.yaml wildfly/wildfly
+```
+It is possible to use a specific https://github.com/wildfly/quickstart/tags[quickstart tag] in case the user does not want to use the `main` branch. To do so, add the following option to the above command:
+```
+--set build.ref={WildFlyQuickStartRepoTag}
+```
+Take a look at the `bootable-jar-openshift` profile configuration in the `todo-backend` quickstart's https://github.com/wildfly/quickstart/blob/main/todo-backend/pom.xml#L254-L304[pom.xml]. The Maven profile named `bootable-jar-openshift` is used by the Helm chart to provision the server with the quickstart deployed. Notice the `wildfly-jar-maven-plugin` configuration defined in this profile. It specifies the layer that should be used when provisioning WildFly. For more details about this, take a look at the https://github.com/wildfly/quickstart/tree/main/todo-backend#architecture[Architecture] section in the quickstart's README.
+
+We have named this application `todo-backend`. This may be changed to a different name, if desired.
+
+The application will now begin to build, and can be observed via:
```

Comment:
"The application will now begin to build. The build can be observed via:"

Otherwise the subject for the 'can be observed' verb is "The application" which feels a bit off.